marcdomain/Fast-Food-Fast
# #161247624 Ft fetch specific order

### What does this PR do?
- Implementation of fetching specific order API endpoint
### Description of Task to be completed
- Use fetch API to consume the get-specific-order endpoint
### How should this be manually tested?
- Using a web browser, access https://marcdomain.github.io/Fast-Food-Fast/views/  to Login  as admin _**email**_ admin@gmail.com _**password**_ adminuser
- Navigate to https://marcdomain.github.io/Fast-Food-Fast/views/orders-admin.html  view all orders.
- Click on _**ID**_ in any order row to see the details of that specific order.
### Any background context you want to provide?
- Authentication is required to view specific order
- User information is included in the order details
### What are the relevant pivotal tracker stories?
- [#161247624](https://www.pivotaltracker.com/story/show/161247624)
